client/components/RelatedProducts/Outfit/OutfitProductCard.jsx

```diff
@@ -48,16 +47,25 @@ const useStyles = makeStyles({
     color: '#442C2E'
   }
 });
-const OutfitProductCard = ({ outfitCardData }) => {
+const OutfitProductCard = ({
+  outfitCardData,
+  removeCard
+}) => {
+
   const classes = useStyles();
+
+  const handleXClick = () => {
+    removeCard(outfitCardData.style_id)
+  }
+
```

Comment:
Well done! It's sometimes tricky to get the relevant data into clickHandlers (at least for me), but this works well! I like that removeCard is it's own function too.